ibis/bigquery/client.py

```diff
@@ -277,6 +264,53 @@ def rename_partitioned_column(table_expr, bq_table):
     return table_expr.relabel({partition_field: col})
 
 
+def parse_project_and_dataset(project_id, dataset_id):
+    """Figure out the project id under which queries will run versus the
+    project_id of where the data live as well as what dataset to use.
+
+    Parameters
+    ----------
+    project_id : str
+        A project name
+    dataset_id : str
+        A ``<project_id>.<dataset_id>`` string or just a dataset name
+
+    Returns
+    -------
+    data_project, run_project, dataset : Tuple[str, str, str]
+
+    Examples
+    --------
+    >>> data_project, run_project, dataset = parse_project_and_dataset(
+    ...     'ibis-gbq',
+    ...     'foo-bar.my_dataset'
+    ... )
+    >>> data_project
+    'foo-bar'
+    >>> run_project
```

Comment:
Can we call this `billing_project` instead of `run_project`?

Reply:
Yep.